hypervisor/src/cpu.rs

```diff
@@ -21,6 +21,13 @@ use crate::MpState;
 use thiserror::Error;
 use vm_memory::GuestAddress;
 
+#[derive(Copy, Clone)]
+pub enum CpuVendor {
+    Intel,
+    AMD,
+    Unknown,
```

Comment:
Maybe make this the default, using the new enum default syntax.

Reply:
Fixed. Also made `get_cpu_vendor` to use `CpuVendor::default()` if none matched. Thanks

Reply:
nit: I would feel more comfortable if the default one was the first in the enum.

Reply:
Done as well, thanks.